src/index.html

```diff
@@ -13,6 +13,76 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+    </div>
```

Comment:
According to the checklist, you should add empty lines between multiline sibling blocks of HTML to simplify reading. However, you should not add them between parent and child elements. In this case, the empty lines between the child div elements inside the parent div are not needed. Removing them would improve the readability of your code.